packages/blade/src/components/Typography/Title/Title.tsx

```diff
@@ -63,21 +68,28 @@ const getProps = ({
     props.as = isPlatformWeb ? 'h1' : undefined;
   }
 
+  // override the computed `as` prop if user passed an `as` prop
+  props.as = as || props.as;
   return props;
 };
 
 export const Title = ({
+  as,
   size = 'small',
   type = 'normal',
   contrast = 'low',
   color,
   children,
   testID,
+  textAlign,
   ...styledProps
 }: TitleProps): ReactElement => {
-  const props = getProps({ size, type, contrast, color, testID });
+  const props = getProps({ as, size, type, contrast, color, testID });
+
+  useValidateAsProp({ componentName: 'Title', as: props.as, validAsValues });
```

Comment:
move this up?

Reply:
In here I'm also checking the props.as which can also have the automatic  computed  as prop, we should validate that too.

Reply:
Done.